itest/lnd_channel_backup_test.go

```diff
@@ -214,24 +228,71 @@ func (c *chanRestoreScenario) testScenario(ht *lntest.HarnessTest,
 	// be cpfp'ed in case of anchor commitments.
 	ht.SetFeeEstimate(30000)
 
-	// Now that we have ensured that the channels restored by the backup
-	// are in the correct state even without the remote peer telling us so,
-	// let's start up Carol again.
-	require.NoError(ht, restartCarol(), "restart carol failed")
+	if c.forceCloseTx == nil {
+		// DLP case.
```

Comment:
Nit: Move the comment above the if clause

Reply:
Fixed